docs/docs/API-Reference/api-openai-responses.mdx

```diff
@@ -0,0 +1,303 @@
+---
+title: OpenAI Responses API
+slug: /api-openai-responses
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/Tabs';
+
+Langflow includes an OpenAI-compatible API endpoint that allows you to execute your flows using OpenAI-compatible request and response formats.
+
+This enables you to use existing OpenAI client libraries with minimal code changes - just change the `model` name to your `flow_id` instead of using OpenAI model names like "gpt-4" or "gpt-3.5-turbo".
+Flow IDs can be found on the code snippets on the [**API access** pane](/concepts-publish#api-access) or in a flow's URL.
+
+## Requirements
+
+Your Langflow flow must contain a **ChatInput** component to be compatible with the OpenAI Responses API.
+Flows without this component will return an error. The component types `ChatInput` and `Chat Input` are recognized as chat inputs.
+
+- **Tools**: The `tools` parameter is not yet supported and will return an error if provided.
+- **Flow Requirements**: Flows must contain a ChatInput component.
+- **Model Names**: The `model` field must contain a valid flow ID or endpoint name.
+- **Authentication**: All requests require an API key passed in the `x-api-key` header:
+
+    ```bash
+    curl -X POST \
+      "$LANGFLOW_SERVER_URL/api/v1/responses" \
+      -H "x-api-key: $LANGFLOW_API_KEY" \
+      -H "Content-Type: application/json" \
+      -d '{
+        "model": "your-flow-id",
+        "input": "Hello, how are you?"
+      }'
+    ```
+
+## Request
+
+```
+POST /api/v1/responses
+```
```

Comment:
I don't think you need this. I think you could just start with the example request and then explain the parameters (header & body).

Put all response stuff under a dedicated subsection.